src/math/ColorManagement.js

```diff
@@ -43,19 +43,15 @@ const _vector = /*@__PURE__*/ new Vector3();
 
 function DisplayP3ToLinearSRGB( color ) {
 
-	color.convertSRGBToLinear();
-
-	_vector.set( color.r, color.g, color.b ).applyMatrix3( LINEAR_DISPLAY_P3_TO_LINEAR_SRGB );
-
-	return color.setRGB( _vector.x, _vector.y, _vector.z );
+	// Display P3 uses the sRGB transfer function
+	return color.convertSRGBToLinear().applyMatrix3( LINEAR_DISPLAY_P3_TO_LINEAR_SRGB );
```

Comment:
TBH, in a future PR, I would prefer to see `color.convertDisplayP3ToLinear()` added. As implemented, this looks like a bug -- in spite of the comment.

Reply:
I'd prefer to wait and see here... I'm concerned `convertXToY` may get cluttered. We could also consider...

```js
ColorManagement.convert( color, DisplayP3ColorSpace, LinearSRGBColorSpace );
```

...to be a public-facing API, perhaps.

Reply:
OK. :-)

Reply:
(I agree it was and is confusing to read! 😅)